### DISCUSSION TITLE
Is it possible to show a new list of suggestions once the value from the first list of suggestions is selected?

### DISCUSSION THREAD

**Discussion by papiya130** [TarekRaafat/autoComplete.js#279] [Q&A]
Is it possible to show a new list of suggestions once the value from the first list of suggestions is selected?
So the scenario is:
once the input is filled, show a list of suggestions. Once we select one of the suggestions, I will make an API call and based on the response I will have to populate the search suggestions again in the same field. Can this be achieved?

**Comment by folknor** [comment]
Yes, you simply use https://tarekraafat.github.io/autoComplete.js/#/configuration?id=startquery
